ncm-metaconfig/src/main/metaconfig/ssh/tests/profiles/allopts.pan

```diff
@@ -0,0 +1,88 @@
+object template allopts;
```

Comment:
rename the template to `client_allopts` (while awaintg your PR for `server_allopts` :smile: